libbeat/processors/translate_ldap_attribute/translate_ldap_attribute.go

```diff
@@ -90,7 +91,15 @@ func (p *processor) String() string {
 }
 
 func (p *processor) Run(event *beat.Event) (*beat.Event, error) {
+	p.log.Debugw("run ldap translation", logp.Stringer("processor", p))
```

Comment:
It looks like this context is used in most (all?) logger calls. Consider storing that in the logger instance, e.g. `.With(logp.Stringer("processor", p))`.